src/sagemaker_pytorch_container/training.py

```diff
@@ -96,6 +96,13 @@ def train(training_environment):
         capture_error = False
         logger.info(f'capture_error is {capture_error}. Default is True')
 
+    try:
+        import torch
+
+        os.environ["SM_DLC_TORCH_VERSION"] = torch.__version__
+    except Exception:
```

Comment:
can we catch some exceptions here? like import error or module not found error.
Also indicate torch is not found in error msg.